pkg/virt-api/rest/subresource_test.go

```diff
@@ -384,7 +384,7 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 			app.RestartVMRequestHandler(request, response)
 
 			Expect(response.Error()).To(HaveOccurred())
-			Expect(response.StatusCode()).To(Equal(http.StatusBadRequest))
+			Expect(response.StatusCode()).To(Equal(http.StatusForbidden))
```

Comment:
We also want to check that `response.Error().Error()` really is the string which we expected.

Reply:
Added.